### PR TITLE
added sequence diagram for ta automation

### PR DESCRIPTION
**Pull request by tnickelsen**

### CODE REVIEW

File: doc/architecture/domains/transfer_agreement_automation.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    receiver ->>+ ta: POST api/transfer-agreement/create-wallet-deposit-endpoint (OBS! should be moved to a wallet backend in the future)
+
+    ta ->>+ wallet: gRPC CreateWalletDepositEndpoint()
+    wallet -->>- ta: WalletDepositEndpoint
+
+    ta -->>- receiver: base64 WalletDepositEndpoint
+
+    receiver ->> sender: base64 WalletDepositEndpoint (over coffee)
+
+    sender ->>+ ta: POST api/transfer-agreement/create
```
Comment (ckr123) [inline doc/architecture/domains/transfer_agreement_automation.md:24]: I dont think we should have a post endpoint call create. could we expand the existing one to take a TA or a base64 string?

File: doc/architecture/domains/transfer_agreement_automation.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+```mermaid
+sequenceDiagram
+    participant robot as TA BackgroundService
+    participant senderWallet as Sender Wallet
+
+    robot ->> robot: GetTransferAgreements
+
+    loop over transfer agreements
+        robot ->> robot: Get owner and make JWT
+        robot ->> senderWallet: gRPC QueryCertificates() with owner JWT
```
Comment (ckr123) [inline doc/architecture/domains/transfer_agreement_automation.md:46]: Could we be more specific that these comes in bulk? Will the endpoint be able to take batch requests?

Comment (tnickelsen) [reply doc/architecture/domains/transfer_agreement_automation.md:46]: In first version we are not sending certificates in bulk

File: doc/architecture/domains/transfer_agreement_automation.md (hunk 1 above)
Comment (tnickelsen) [reply doc/architecture/domains/transfer_agreement_automation.md:24]: This is not ment to be the exact name for the call, but sure I can rename it

**Review by ckr123 — APPROVED** [review]

**Pull request merged**